kubernetes_platform/proto/kubernetes_executor_config.proto

```diff
@@ -122,3 +124,15 @@ message PodMetadata {
     map<string, string> labels = 1;
     map<string, string> annotations = 2;
 }
+
+message ConfigMapAsVolume {
+    // Name of the ConfigMap.
+    string config_map_name = 1;
+    // Container path to mount the ConfigMap data.
+    string mount_path = 2;
+}
+
+message ConfigMapAsEnv {
+    // Name of the ConfigMap.
+    string config_map_name = 1;

```

Comment:
Since Kubernetes support mounting configmap with key reference, we should also support that in the protobuf definition just like the secret as env at line 44-49. (e.g.)

```
    message ConfigMapKeyToEnvMap {
        // Corresponds to a key of the ConfigMap.data field.
        string configmap_key = 1;
        // Env var to which configmap_key's data should be set.
        string env_var = 2;
    }
```